app/components/dex-create.jsx

```diff
@@ -7,14 +7,29 @@ import slug          from 'slug';
 import { AlertComponent } from './alert';
 import { ReactGA }        from '../utils/analytics';
 import { createDex }      from '../actions/dex';
+import { listGames }      from '../actions/games';
 
 const NATIONAL_ONLY_GAMES = ['x', 'y', 'omega_ruby', 'alpha_sapphire'];
 
 export class DexCreate extends Component {
 
   constructor (props) {
     super(props);
-    this.state = { error: null, game: 'sun', regional: false };
+    this.state = { error: null, game: 'ultra_moon', games: [], regional: false };
+  }
+
+  componentWillMount () {
+    this.reset();
+  }
+
+  reset (props) {
+    const { listGames } = props || this.props;
+
+    listGames()
+    .then((games) => {
+      games.sort((a, b) => b.order - a.order);
```

Comment:
hmm we shouldnt need to order on the front end, but i see why you did it. ill see if i can do that on the backend

Reply:
this has been deployed to api, so you can remove all sorting lines